card/mod/01_core/spec/set/all/tracked_attributes_spec.rb

```diff
@@ -115,107 +115,112 @@ def cards_should_be_added number
     # Can't we just move this data to fixtures?
 
 
-    it "renaming plus card to its own child" do
-      assert_rename card("A+B"), "A+B+T"
+    it 'renaming plus card to its own child' do
+      assert_rename card('A+B'), 'A+B+T'
     end
 
-    it "clears cache for old name" do
+    it 'clears cache for old name' do
       assert_rename Card['Menu'], 'manure'
       expect(Card['Menu']).to be_nil
     end
 
-    it "wipes old references by default" do
+    it 'wipes old references by default' do
       c = Card['Menu']
       c.name = 'manure'
       c.save!
       expect(Card['manure'].references_from.size).to eq(0)
     end
 
-    it "picks up new references" do
+    it 'picks up new references' do
       Card.create name: 'kinds of poop', content: '[[manure]]'
       assert_rename Card['Menu'], 'manure'
       expect(Card['manure'].references_from.size).to eq(2)
     end
 
-    it "handles name variants" do
-      assert_rename card("B"), "b"
+    it 'handles name variants' do
+      assert_rename card('B'), 'b'
     end
 
-    it "handles plus cards renamed to simple" do
-      assert_rename card("A+B"), "K"
+    it 'handles plus cards renamed to simple' do
+      assert_rename card('A+B'), 'K'
     end
 
 
-    it "handles flipped parts" do
-      assert_rename card("A+B"), "B+A"
+    it 'handles flipped parts' do
+      assert_rename card('A+B'), 'B+A'
     end
 
-    it "test_should_error_card_exists" do
-      @t=card("T"); @t.name="A+B";
-      assert !@t.save, "save should fail"
-      assert @t.errors[:name], "should have errors on key"
+    it 'test_should_error_card_exists' do
+      @t=card('T'); @t.name='A+B';
+      assert !@t.save, 'save should fail'
+      assert @t.errors[:name], 'should have errors on key'
     end
 
-    it "test_used_as_tag" do
-      @b=card("B"); @b.name='A+D'; @b.save
+    it 'test_used_as_tag' do
+      @b=card('B'); @b.name='A+D'; @b.save
```

Comment:
Do not use semicolons to terminate expressions.<br>Surrounding space missing for operator `=`.